Buckify backends/arm for meta internal use.

hsharma35 opened the pull request:
Summary:
1. Add buck targets for `executorch/backends/arm/...`
2. Pyre typing cleanup
3. Invoke vela compiler as `vela.main(args)` instead of `subprocess.run([vela_command]...)`

Differential Revision: D62062674

digantdesai reviewed (approved):
Running some more jobs. Make sure `arm` jobs are clean. Stamping it.

hsharma35 committed:
Buckify backends/arm for meta internal use. (#5023)

Summary:
Pull Request resolved: https://github.com/pytorch/executorch/pull/5023

1. Add buck targets for `executorch/backends/arm/...`
2. Pyre typing cleanup
3. Invoke vela compiler as `vela.main(args)` instead of `subprocess.run([vela_command]...)`

Differential Revision: D62062674
